exampleCourse/questions/stringOperations/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "c07e099f-0149-471e-9c44-ab55622bedec",
+    "title": "String Operations",
```

Comment:
Can we name this question `String answers with pl-string-input` and add this question to homework4 (which is questions showing how to use different elements)?

Reply:
@mfsilva22 also, could you update folder name from `stringOperations` to `examplesStringInput`. This allows for questions configured to show _all_ the possible features of an element.

The HW @mwest1066 is referring to can be found here:

https://github.com/PrairieLearn/PrairieLearn/blob/37b150c83341d66e499ddb8afd05b348323550b9/exampleCourse/courseInstances/Sp15/assessments/plelementexamples/infoAssessment.json#L36

Note: Naming conventions will be updated later this weekend in #1069